libs/libarchfpga/src/arch_util.h

```diff
@@ -15,6 +15,8 @@ void set_arch_file_name(const char* arch);
  */
 const char* get_arch_file_name();
 
+const std::string EMPTY_BLOCK_NAME("EMPTY");
```

Comment:
Currently, most constant variable is defined by ``constexpr``. And most of time, we use ``char*`` than ``std::string``, to save memory.
This is a really preference from my side.
@hzeller Would you mind to share your opinions here?

Reply:
No strong opinion on my side actually, I have set this to be `constexpr`